wallet/internal/txsizes/size.go

```diff
@@ -78,13 +78,13 @@ const (
 	P2PKHOutputSize = 8 + 2 + 1 + 25
 )
 
-// EstimateSerializeSize returns a worst case serialize size estimate for a
-// signed transaction that spends a number of outputs and contains each
-// transaction output from txOuts. The estimated size is incremented for an
-// additional change output if changeScriptSize is greater than 0. Passing 0
-// does not add a change output.
+// EstimateSerializeSize returns a worst case serialize size
+// estimate for a signed transaction that spends a number of outputs and
+// contains each transaction output from txOuts. The estimated size is
+// incremented for an additional change output if changeScriptSize is greater
+// than 0. Passing 0 does not add a change output.
```

Comment:
Nothing changed here except the wrapping.  Can this be reverted? (please inspect diffs for trivial changes before submitting!)

Reply:
My bad, updating my workflow. Will improve.